DiffPlex.Wpf/Controls/InternalLinesViewer.xaml.cs

```diff
@@ -68,7 +70,151 @@ public void Clear()
         ValuePanel.Children.Clear();
     }
 
-    public StackPanel Add(int? number, string operation, string value, string changeType, UIElement source)
+    public WrapPanel Add(int? number, string operation, string value, string changeType, UIElement source)
```

Comment:
There is a lot of duplications between the wrap and nowrap versions of these methods, can we clean that up?

Reply:
This has been pushed

Reply:
This has been reverted. The clean up for some reason slowed the code down. I separated each operation into it's own method and simply checked if wrap or no wrap from a singular method. But there seems to be a loop that goes on for over 5 minutes creating all the textblocks

Reply:
This has been fixed and pushed